packages/esm-system-admin-app/src/change-password-modal/change-password-modal.component.tsx

```diff
@@ -0,0 +1,197 @@
+        } else if (!lowercasePassword) {
+          errMsg = t('atLeastOneLowercase', 'At least one Lowercase');
+        } else if (!digitsPassword) {
+          errMsg = t('atLeastOneDigit', 'At least one digit');
+        } else if (!minLengthPassword) {
+          errMsg = t('minimum8Characters', 'Minimum 8 characters');
+        } else if (passwordInput.oldPassword.length > 0 && passwordInput.newPassword === passwordInput.oldPassword) {
+          errMsg = t('newPasswordMustNotBeTheSameAsOld', 'New password must not be the same as password');
+        } else {
+          errMsg = '';
+          setIsNewPasswordInvalid(false);
+        }
+        setNewPasswordErr(errMsg);
+      } else if (
+        passwordInputFieldName === 'confirmPassword' ||
+        (passwordInputFieldName === 'newPassword' && passwordInput.confirmPassword.length > 0)
+      ) {
+        if (passwordInput.confirmPassword !== passwordInput.newPassword) {
+          setConfirmPasswordError(t('confirmPasswordMustBeTheSameAsNew', 'Confirm password must be the same as new'));
+        } else {
+          setConfirmPasswordError('');
+          setIsConfirmPasswordInvalid(false);
+        }
+      } else {
+        if (passwordInput.newPassword.length > 0 && passwordInput.newPassword === passwordInput.oldPassword) {
+          setOldPasswordErr(t('oldPasswordMustNotBeTheSameAsNew', 'Old password must not be the same as new'));
+        } else {
+          setOldPasswordErr('');
+          setIsOldPasswordInvalid(false);
+        }
+      }
+    },
+    [passwordInput.confirmPassword, passwordInput.newPassword, passwordInput.oldPassword, t],
+  );
+
+  useEffect(() => {
+    if (passwordInput.oldPassword !== '') {
+      handleValidation(passwordInput.oldPassword, 'oldPassword');
+    }
+    if (passwordInput.newPassword !== '') {
+      handleValidation(passwordInput.newPassword, 'newPassword');
+    }
+    if (passwordInput.confirmPassword !== '') {
+      handleValidation(passwordInput.confirmPassword, 'confirmPassword');
+    }
+  }, [handleValidation, passwordInput]);
+
+  const handlePasswordChange = (event) => {
+    const passwordInputValue = event.target.value.trim();
+    const passwordInputFieldName = event.target.name;
+    const NewPasswordInput = { ...passwordInput, [passwordInputFieldName]: passwordInputValue };
+    setPasswordInput(NewPasswordInput);
+  };
+
+  const handleSubmit = useCallback(
+    async (evt: React.FormEvent<HTMLFormElement>) => {
+      evt.preventDefault();
+      setIsSavingPassword(true);
+      performPasswordChange(passwordInput.oldPassword, passwordInput.confirmPassword)
+        .then(() => {
+          close();
+          navigate({ to: `\${openmrsSpaBase}/logout` });
+          showSnackbar({
+            isLowContrast: true,
+            kind: 'success',
+            subtitle: t('userPasswordUpdated', 'User password updated successfully'),
+            title: t('userPassword', 'User password'),
```

Comment:
```suggestion
            title: t('passwordChanged', 'Password changed'),
						subtitle: t('passwordUpdatedSuccessfully', 'Your password has been successfully updated.')
```

or something to that ewffect.